cmd/functions.go

```diff
@@ -156,7 +158,7 @@ Enter your project ref: `, utils.GetSupabaseDashboardURL())
 
 		scanner := bufio.NewScanner(os.Stdin)
 		if !scanner.Scan() {
-			return errors.New("Cancelled " + utils.Aqua("supabase functions deploy") + ".")
+			return errors.New("Cancelled " + utils.Aqua(cmd.UseLine()) + ".")
```

Comment:
this will print out the `Usage` information of the command - for example, cancelling the `supabase sso info ...` will print `Cancelled supabase sso info [flags]`